webapp/channels/src/components/integrations/add_command/add_command.tsx

```diff
@@ -58,7 +58,6 @@ const AddCommand = ({team, actions}: Props) => {
             header={HEADER}
             footer={FOOTER}
             loading={LOADING}
-            renderExtra={''}
```

Comment:
Was this unused property?

Reply:
so, instead of passing the empty string, I just removed as it is of type JSX element